phpBB/phpbb/user_loader.php

```diff
@@ -193,7 +194,7 @@ public function get_avatar($user_id, $query = false)
 			include($this->phpbb_root_path . 'includes/functions_display.' . $this->php_ext);
 		}
 
-		return get_user_avatar($user['user_avatar'], $user['user_avatar_type'], $user['user_avatar_width'], $user['user_avatar_height']);
+		return get_user_avatar($user['user_avatar'], $user['user_avatar_type'], $user['user_avatar_width'], $user['user_avatar_height'], 'USER_AVATAR', false, $lazy);
```

Comment:
Please phpbb_get_avatar() here. NO idea why get_user_avatar() is being used as that is a legacy function ...